app/controllers/cookies/policies_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Cookies
+  class PoliciesController < ApplicationController
+    before_action { @no_shared_search = true }
+
+    def show
+      @policy = cookies_policy
```

Comment:
```suggestion
      @cookies_policy = cookies_policy
```

Reply:
This will actually clash with the @cookies_policy set in `ApplicationController#cookies_policy` helper method - since this is duplicating that assignment anyway I've replaced references to `@policy` with usage of the helper_method `cookies_policy`